openrarity/collection/collection.py

```diff
@@ -169,6 +223,18 @@ def rank_collection(
         """Preprocess tokens then rank the tokens for the collection and set the
         corresponding cls.ranks attribute. Optionally, return the ranks.
 
+        Notes
+        -----
+        metric.probability : float
+            It is the Product of minimum value of (1,sum of `token.supply` values divided by `total_supply` value) of all tokens
+        metric.information : float
+            It is the Sum of maximum value of (0, -log2(sum of `token.supply` values divided by `total_supply` value)) of all tokens
+        metric.information_entropy : float
+            It is the division value of `metric.information` and `entropy`
+        metric.unique_trait_count : int
+            It is the sum of unique `token_count` values
+        metric.max_trait_information : float
+            It is the maximum value of attribute statistics `metric.information`
```

Comment:
Don't say what the code is doing. Describe in laymans terms...
```python
"""
     metric.probability : float
            The statistical probability/likelihood that a token exists in a collection
"""
```
Don't mention things like `min(1, sum of token.supply)`.

Reply:
fixed it.